include/tdnftypes.h

```diff
@@ -166,7 +166,7 @@ typedef struct _TDNF_PKG_INFO
     char* pszFormattedDownloadSize;
     char* pszRelease;
     char* pszLocation;
-    char **ppszDependencies;
+    char ***ppszDependencies;
```

Comment:
Should be renamed to `pppszDependencies` (three `p`s)

Reply:
Done